README.md

```diff
@@ -100,7 +125,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+Schema describes the columns name, data type, any constraint of database. If there is foreign key in the element then it's one-to-many relationship.
+
+For example, we can have person table and wishes table. Person has person_id and wishes could have person_id as foreign key.
 ```
```

Comment:
Foreign keys aren't limited to one-to-many relationships. We'll talk more about this during the lesson on has_many :through